techlottus/portalverse-uane
# [DEV-5255] Migrate "extension universitaria" pages images

## Issue
 - Migrate "extension universitaria" pages images from drive to S3

## Solution
 - [feat: extension universitaria pages images migrated](https://github.com/techlottus/portalverse-uane/commit/d656a4863ba93c28146e5e7b5849dfcc1e9952a6)

## Testing
 - Go to `/extension-universitaria`
 - Select any course 
 - Watch the page and check the URL from images (assets.staging.bedu.org)
 - Keep rocking 🔥
